.packit.yml

```diff
@@ -44,3 +24,61 @@ jobs:
   - fedora-rawhide-ppc64le
   - fedora-rawhide-s390x
   - fedora-rawhide-x86_64
+
+  actions:
+    post-upstream-clone:
+      # Use the Fedora Rawhide specfile
+      - "git clone https://src.fedoraproject.org/rpms/systemd .packit_rpm --depth=1"
+      # Drop the "sources" file so rebase-helper doesn't think we're a dist-git
+      - "rm -fv .packit_rpm/sources"
+      # Drop backported patches from the specfile, but keep the downstream-only ones
+      # - Patch(0000-0499): backported patches from upstream
+      # - Patch0500-9999: downstream-only patches
+      - "sed -ri '/^Patch(0[0-4]?[0-9]{0,2})?\\:.+\\.patch/d' .packit_rpm/systemd.spec"
+      # Build the RPM with --werror. Even though --werror doesn't work in all
+      # cases (see [0]), we can't use -Dc_args=/-Dcpp_args= here because of the
+      # RPM hardening macros, that use $CFLAGS/$CPPFLAGS (see [1]).
+      #
+      # [0] https://github.com/mesonbuild/meson/issues/7360
+      # [1] https://github.com/systemd/systemd/pull/18908#issuecomment-792250110
+      - 'sed -i "/^CONFIGURE_OPTS=(/a--werror" .packit_rpm/systemd.spec'
+      # Ignore unpackages standalone binaries
+      - "sed -i 's/assert False,.*/pass/' .packit_rpm/split-files.py"
+
+- job: copr_build
+  trigger: pull_request
```

Comment:
I'd recommend also doing `commit` (or doing only `commit` if you don't care about having a build for every PR). For the `commit` ones, you probably want to also set `owner` and `project` to they always get delivered into the same copr repo (instead of making a new temporary one every time). We already have a group for Hyperscale at https://copr.fedorainfracloud.org/groups/g/centoshyperscale/coprs/ we can use for this.

Reply:
Will switch to commit once this is working, need pull_request atm to test the PR

Reply:
Also switched to custom hyperscale owner and copr projects